index.html

```diff
@@ -33,6 +33,151 @@ <h3>LET’S CONNECT</h3>
+          <img class="snapshoot" src="images/SnapshootD2.jpg" alt="Project-2 Snapshoot">
+        </div>
+        <div class="projects">
+          <h2 class="project-titles">Multi-Post Stories</h2>
+          <ul class="inline-li project-summary">
+            <li class="client">CANOPY</li>
+            <li class="type">Back End Dev</li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="project-descriptions">
+            Experimental content creation feature that allows users to add to an existing 
+            story over the course of a day without spamming their friends.
+            </p>
+          <ul class="inline-li">
+            <li class="tags">html</li>
+            <li class="tags">css</li>
+            <li class="tags">javascripts</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="cards" id="card3">
+        <div class="snapshoot-placeholders">
+          <img class="snapshoot" src="images/SnapshootD3.jpg" alt="Project-1 Snapshoot">
+        </div>
+        <div class="projects">
+          <h2 class="project-titles">Tonic</h2>
+          <ul class="inline-li project-summary">
+            <li class="client">CANOPY</li>
+            <li class="type">Back End Dev</li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="project-descriptions">
+            A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+          </p>
+          <ul class="inline-li">
+            <li class="tags">html</li>
+            <li class="tags">css</li>
+            <li class="tags">javascripts</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="cards reverse" id="card4">
+        <div class="snapshoot-placeholders">
+          <img class="snapshoot" src="images/SnapshootD4.jpg" alt="Project-2 Snapshoot">
+        </div>
+        <div class="projects">
+          <h2 class="project-titles">Multi-Post Stories</h2>
+          <ul class="inline-li project-summary">
+            <li class="client">CANOPY</li>
+            <li class="type">Back End Dev</li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="project-descriptions">
+            Experimental content creation feature that allows users to add to an existing 
+            story over the course of a day without spamming their friends.
+            </p>
+          <ul class="inline-li">
+            <li class="tags">html</li>
+            <li class="tags">css</li>
+            <li class="tags">javascripts</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="flex" id="about">
+      <div class="snapshoot-placeholders">
+        <h2 class="head-1" id="about-title">About<br> 
+          Myself</h2>
```

Comment:
- [ ] Please make sure the `font-size` of your 'About us' heading is similar to the `font-size` of [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=tG1jh7hS4l814OsZ-0). Your design should stick to the Figma design as closely as possible according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)


**_Figma_design_**

<img width="275" alt="Screenshot 2023-02-17 at 12 03 29 PM" src="https://user-images.githubusercontent.com/44798044/219585919-0f42fa88-d20e-4be2-8a0d-0080bae73350.png">


_**Your design**_ 

<img width="253" alt="Screenshot 2023-02-17 at 12 03 23 PM" src="https://user-images.githubusercontent.com/44798044/219585859-b9a9678f-0ba4-4ffd-a22a-623cb929fdba.png">

Reply:
Done